.travis.yml

```diff
@@ -10,7 +10,6 @@ install:
 
 script:
   - PYTHONWARNINGS=once coverage run --omit=*.virtualenvs*,*virtualenv* src/manage.py test --settings=sous_chef.settings_test member meal order notification delivery note billing page
-  - python3 src/manage.py test --settings=sous_chef.settings_test_fr member meal order notification delivery note billing page
```

Comment:
I intentionally added this to test the application in a French setting. I think there were some differences in string comparison. Why do we delete this line?